cvat-ui/src/actions/tasks-actions.ts

```diff
@@ -7,42 +7,60 @@ import getCore from '../core';
 const cvat = getCore();
 
 export enum TasksActionTypes {
+    GET_TASKS = 'GET_TASKS',
     GET_TASKS_SUCCESS = 'GET_TASKS_SUCCESS',
     GET_TASKS_FAILED = 'GET_TASKS_FAILED',
+    LOAD_ANNOTATIONS = 'LOAD_ANNOTATIONS',
+    LOAD_ANNOTATIONS_SUCCESS = 'LOAD_ANNOTATIONS_SUCCESS',
+    LOAD_ANNOTATIONS_FAILED = 'LOAD_ANNOTATIONS_FAILED',
+    DUMP_ANNOTATIONS = 'DUMP_ANNOTATIONS',
+    DUMP_ANNOTATIONS_SUCCESS = 'DUMP_ANNOTATIONS_SUCCESS',
+    DUMP_ANNOTATIONS_FAILED = 'DUMP_ANNOTATIONS_FAILED',
 }
 
-export function getTasksSuccess(array: any[], previews: string[],
-    count: number, query: TasksQuery): AnyAction {
+function getTasks(): AnyAction {
+    const action = {
+        type: TasksActionTypes.GET_TASKS,
+        payload: {},
+    };
+
+    return action;
+}
+
+function getTasksSuccess(array: any[], previews: string[],
+    count: number, gettingQuery: TasksQuery): AnyAction {
     const action = {
         type: TasksActionTypes.GET_TASKS_SUCCESS,
         payload: {
             previews,
             array,
             count,
-            query,
+            gettingQuery,
         },
     };
 
     return action;
 }
 
-export function getTasksFailed(error: any, query: TasksQuery): AnyAction {
+function getTasksFailed(tasksFetchingError: any, gettingQuery: TasksQuery): AnyAction {
     const action = {
         type: TasksActionTypes.GET_TASKS_FAILED,
         payload: {
-            error,
-            query,
+            tasksFetchingError,
```

Comment:
Again, why do you need to rename these variables? Names were perfect, short, and meaningful.

Reply:
Returned this name here